src/Google/Ads/GoogleAds/Lib/V1/GoogleAdsClientBuilder.php

```diff
@@ -242,6 +243,12 @@ public function validate()
                 'OAuth2 authentication credentials must not be null.'
             );
         }
+
+        if (is_null($this->logLevel)) {
+            $this->logLevel = LogLevel::INFO;
+        } elseif (!defined('Psr\Log\LogLevel::' . $this->logLevel)) {
```

Comment:
Will this work with the case when people specify small letters as well?

Reply:
It didn't because I wasn't sure we should allow lowercase values for the log level.

However, it was easy to allow them, so I just added support for that.